modules/blocks/registration.js

```diff
@@ -10,24 +10,29 @@ const blocks = {};
  *
  * @param {string} slug     Block slug
  * @param {Object} settings Block settings
+ * @return {Boolean}        Whether the block has been successfuly registered
```

Comment:
In my original project scaffolding, I had this function return the block which had been registered, thinking we might apply some default values and it'd be useful for whoever registering to gain access to them. Do you think we could instead return a `{?WPBlock}` type (`return;` undefined if failed).